package.json

```diff
@@ -83,9 +78,7 @@
     "semantic-release": "17.4.7",
     "tailwindcss": "2.2.19",
     "ts-jest": "27.0.7",
-    "ts-loader": "8.3.0",
-    "typescript": "4.3.5",
-    "webpack": "4.46.0"
```

Comment:
a lot of dependencies are not needed. they're likely implicit with the new storybook version